lib/Classifiers/Images/ClusteringFaceClassifier.php

```diff
@@ -55,15 +55,26 @@ public function classify(array $queueFiles): void {
 			$timeout = self::IMAGE_TIMEOUT;
 		}
 
-		// Only process files that haven't been processed before
-		$queueFiles = array_values(
-			array_filter($queueFiles, fn($queueFile) =>
-				count($this->faceDetections->findByFileId($queueFile->getFileId())) === 0
-			)
-		);
+		$filteredQueueFiles = [];
+		foreach ($queueFiles as $queueFile) {
+			try {
+				$filesCount = count($this->faceDetections->findByFileId($queueFile->getFileId()));
+			} catch (Exception $e) {
+				$filesCount = 1;
+			}
+			if ($filesCount !== 0) {
+				try {
+					$this->queue->removeFromQueue(self::MODEL_NAME, $queueFile);
```

Comment:
The logic here filters out files that have detections already To avoid classifying the same file twice. If a file has the same face twice it will be handled elsewhere (only the face vector that matches the others in the cluster most closely will be kep in the db).

Reply:
If you want to reclassify files, this PR adds a reset button and a reset command so that you can get new classifications and new clusters.